draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -214,8 +214,6 @@ However, the term "salt" is used throughout this document for brevity.
 
 `SD-CLAIMS` can also be nested deeper to capture more complex objects, as will be shown later.
 
-`SD-JWT` is sent from the Issuer to the Holder, together with the mapping of the plain-text claim values, the salt values, and potentially some other information.
```

Comment:
Was this perceived redundant?

Reply:
Yeah, it felt redundant and somewhat out of order with the content just below.